HPXMLtoOpenStudio/resources/hpxml_schematron/EPvalidator.xml

```diff
@@ -240,6 +240,7 @@
       <sch:assert role='ERROR' test='count(h:Systems/h:HVAC/h:HVACPlant/h:HeatingSystem) &gt;= 0'>Expected 0 or more element(s) for xpath: Systems/HVAC/HVACPlant/HeatingSystem</sch:assert> <!-- see [HeatingSystem] -->
       <sch:assert role='ERROR' test='count(h:Systems/h:HVAC/h:HVACPlant/h:CoolingSystem) &gt;= 0'>Expected 0 or more element(s) for xpath: Systems/HVAC/HVACPlant/CoolingSystem</sch:assert> <!-- see [CoolingSystem] -->
       <sch:assert role='ERROR' test='count(h:Systems/h:HVAC/h:HVACPlant/h:HeatPump) &gt;= 0'>Expected 0 or more element(s) for xpath: Systems/HVAC/HVACPlant/HeatPump</sch:assert> <!-- see [HeatPump] -->
+      <sch:assert role='ERROR' test='count(h:Systems/h:HVAC/h:HVACPlant/h:GeothermalLoop) &gt;= 0'>Expected 0 or more element(s) for xpath: Systems/HVAC/HVACPlant/GeothermalLoop</sch:assert> <!-- see [GeothermalLoop] -->
```

Comment:
Move this to [here](https://github.com/NREL/OpenStudio-HPXML/blob/ff35611236bbfb38d3cd026e3f58e63a19c4f625/HPXMLtoOpenStudio/resources/hpxml_schematron/EPvalidator.xml#L1171).